docs/system/k8s-metrics.md

```diff
@@ -162,6 +200,44 @@ This metric is [recommended][MetricRecommended].
 <!-- markdownlint-disable -->
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `k8s.node.memory.usage`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.k8s.node.memory.usage(metric_table) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `k8s.node.memory.usage` | Gauge | `By` | Memory usage of the Node [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+**[1]:** Total memory usage of the Node
+
+
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+<!-- semconv metric.k8s.node.memory.usage -->
```

Comment:
you no longer need to provide BOTH of these semconv snippet generation bits.   Now - any attributes will automatically be included in the same snippet.